core/src/main/java/com/salesforce/slds/shared/models/core/Entry.java

```diff
@@ -141,17 +147,17 @@ public boolean hasAnnotations() {
                 if (endColumn == -1) {
                     // we found a sldsValidatorIgnore with no sldsValidatorAllow after it
                     // so we should exempt the rest of the lines.
-                    lintingIgnoreRanges.add(
+                    this.recommendationSuppressionRanges.add(
                             new Range(
                                     new Location(i, startColumnIgnore),
-                                    new Location(Integer.MAX_VALUE, Integer.MAX_VALUE)
+                                    new Location(i, Integer.MAX_VALUE)
```

Comment:
Is changing `Integer.MAX_VALUE` to `i` intentional? From what I recall (rather vaguely) the point here was to figure out the range that suppression should be applied to.

For example, if we had a doc with 100 lines and at line `i=50` we encountered `sldsValidatorIgnore` but in lines 51-100 there were no mention of `sldsValidatorAllow` then we would want to suppress warnings in lines 50-100 (which I believe is consistent with ESLint rules).

But with this change here, we would only ignore lines 50-50 instead.

Reply:
In the previous version, annotation applied to the line only. Let me play with it to see what the syntax would look like. Thanks.

Reply:
Here is what I'm thinking for CSS, let me know what you think

```css
/* following ESLint Pattern */
/* @sldsValidatorIgnore */ .THIS { padding: 0; } /* @sldsValidatorAllow */

.THIS { /* @sldsValidatorIgnoreNextLine */ 
padding: 0;
padding: 0;

/*CSS Lint Pattern for inline ignore */
.THIS { /* @sldsValidatorIgnore */ padding: 0
padding: 0; }

```